api/V1/User.js

```diff
@@ -158,4 +164,25 @@ module.exports = function(app, baseUrl) {
       });
     })
   );
+
+  /**
+   * @api {get} api/v1/endUserAgreementVersion Get the latest end user agreement version
+   * @apiName EndUserAgreementVersion
+   * @apiGroup User
+   *
+   * @apiSuccess {Integer} euaVersion Version of the latest end user agreement.
+   * @apiUse V1ResponseSuccess
+   *
+   * @apiUse V1ResponseError
+   */
+  app.get(
+    baseUrl + "/endUserAgreementVersion",
```

Comment:
I can imagine that we might end up having the EUA text stored in the API server so perhaps leave room for this by making the URL `/api/v1/endUserAgreement/latest`